recipes/b2b-utils/meta.yaml

```diff
@@ -0,0 +1,96 @@
+{% set name = "b2b-utils" %}
+{% set version = "0.017" %}
+{% set sha256 = "99fe213d6404076b5451f441413b628df3357ddb1c9e9c5458974addac02d6f0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/jvolkening/b2b-utils/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  host:
+    - make
+    - perl
+    - perl-bioperl
+    - perl-biox-seq >=0.008006
+    - perl-dbi
+    - perl-file-compare
+    - perl-list-moreutils
+    - perl-module-build
+    - perl-perlio-gzip
+    - perl-test-more
+    - perl-test-pod
+    - bwa
+    - mafft
+    - medaka
+    - miniasm
+    - minimap2
+    - samtools
+  run:
+    - perl
+    - perl-bioperl
+    - perl-biox-seq >=0.008006
+    - perl-dbi
+    - perl-list-moreutils
+    - perl-perlio-gzip
+
+test:
+  commands:
+    - bam2consensus --version
+    - blast2diamond --version
+    - boot2frac --version
+    - decode_sam_flag --version
+    - fasta2nexus --version
+    - fasta2phylip --version
+    - fq_deinterleave --version
+    - fq_interleave --version
+    - frag_lens --version
+    - gb2fasta --version
+    - guess_ill_instrument --version
+    - minimeta --version
+    - ncbi_blast_update --version
+    - qbin --version
+    - rm_chim --version
+    - rm_gaps --version
+    - seq_diff --version
+    - summarize_assembly --version
+    - sync_reads --version
+    - uc --version
+
+about:
+  home: https://github.com/jvolkening/b2b-utils
+  dev_url: https://github.com/jvolkening/b2b-utils
+  license: GPLv3
+  license_family: GPL
+  license_file: LICENSE
+  summary: Genomics tools from BASE2BIO
+  description: |
+    This package contains a set of programs and utilities for working with
+    genomic data. Please see the in-program documentation (`--help`) of each
+    individual tool for usage and details. Please note that some tools in this
+    suite utilize dependencies that are not explicity stated in the Conda
+    package. This keeps the install footprint smaller since all users will not
+    use all utilities, but if you receive error messages about missing
+    programs you will need to install those as well. All possible
+    dependencies should be available as Conda packages.
+
+extra:
+  recipe-maintainers:
+    - jvolkening
+  skip-lints:
+    - should_be_noarch_generic
```

Comment:
why is this tool not noarch: generic?

Reply:
Its not really. You can try and make sure the package is coming from conda-forge: https://github.com/conda-forge/perl-dbi-feedstock/blob/main/recipe/meta.yaml

Reply:
If you'd like I will update it as noarch and see if it passes CI testing. Based on already trying to test/build with the latest bioconda-utils + docker + mulled-testing, I expect it to fail during mulled build with "DBI.pm not found." I actually already originally tried to solve the failed build by downgrading the perl-dbi version *from* conda-forge *to* the older bioconda package...neither worked. Logs indicated that in both cases, perl-dbi was not in the list of installed conda packages during mulled testing, despite being listed as a requirement, but it *was* listed during the initial successful non-mulled docker build.  Removing `noarch: generic` allowed the build to succeed, without any other changes.

Should I push a `noarch` commit? Thanks.